lib/src/screens/login_screen.dart

```diff
@@ -0,0 +1,259 @@
+                              children: [
+                                Container(
+                                  width: logoIconContainerWidth,
+                                  height: logoIconContainerHeight,
+                                  decoration: BoxDecoration(
+                                      image: DecorationImage(
+                                          image:
+                                              AssetImage(AppLoginScreen.logo))),
+                                ),
+                                Padding(
+                                    padding:
+                                        EdgeInsets.only(left: logoLeftPadding)),
+                                Container(
+                                  width: logoTextContainerWidth,
+                                  height: logoTextContainerHeight,
+                                  decoration: BoxDecoration(
+                                      image: DecorationImage(
+                                          image: AssetImage(
+                                              AppLoginScreen.textLogo))),
+                                ),
+                              ],
+                            ),
+                          ),
+                          Padding(
+                              padding:
+                                  EdgeInsets.only(top: welcomeBackPaddingTop)),
+                          Container(
+                              alignment: Alignment.center,
+                              width: maxWidth,
+                              child: Text(
+                                AppLoginScreen.welcomeBack,
+                                style: TextStyle(
+                                    fontSize: welcomeBackFontSize,
+                                    fontFamily: AppFonts.fontAppRegular),
+                              )),
+                        ],
+                      ),
+                    ),
+                    Center(
+                      child: Container(
+                        child: Center(
+                          child: SingleChildScrollView(
+                            child: Container(
+                              width: loginBodyWidthContainer,
+                              height: loginBodyHeightContainer,
+                              decoration: BoxDecoration(
+                                  color: Colors.white,
+                                  borderRadius:
+                                      BorderRadius.all(Radius.circular(10))),
+                              alignment: Alignment.center,
+                              child: Center(
+                                child: Login_Body(
+                                  Height: loginBodyHeightContainer,
+                                  Width: loginBodyWidthContainer,
+                                ),
+                              ),
+                            ),
```

Comment:
is it too long to read? can you make it shorter?

Reply:
It is not duplicated because the same part of screen i already build in login_body, each part of each device is built differently